### PR TITLE
ROC-7628: Revert the ROC-7612 change introduced in system due send letter client version upgrade

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-7626

### Change description ###

Revert the ROC-7612 change introduced in system due send letter client version upgrade

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java [hunk 1]
```diff
@@ -146,7 +147,8 @@ public void printPdf(Claim claim, List<Printable> documents) {
             return document.template.getBytes();
         }
 
-        return pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
+        byte[] html = pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
```
Comment (timja) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: have you tested this change anywhere?

Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: I have reverted back to old code as it was removed by mistake

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java [hunk 2]
```diff
@@ -146,7 +147,8 @@ public void printPdf(Claim claim, List<Printable> documents) {
...
+        return Base64.getEncoder().encodeToString(html).getBytes();
```
Comment (njrich28) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]:
```suggestion
        return Base64.getEncoder().encode(html);
```

No need to encode to `String` and back to `byte[]` again. Also, `getBytes` uses the platform default character encoding but `encodeToString` always uses ISO-8859-1 so you may run into encoding issues.

Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: yes it is failing , so looking into it

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java (hunk 1 above)
Comment (pragnesh-patel) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: We need tests

**Pull request merged**